obspy/io/seiscomp/data/quakeml_1.2__sc3ml_0.10.xsl

```diff
@@ -103,7 +103,7 @@ Parent              Element lost
 amplitude           evaluationStatus
 magnitude           evaluationMode
 originUncertainty   confidenceLevel
-arrival             commment
+arrival             comment
 arrival             horizontalSlownessWeight
```

Comment:
not sure about these stylesheets, I think some are external documents, some probably done by us, maybe keep them unchanged